lib/beaker-puppet/helpers/puppet_helpers.rb

```diff
@@ -200,11 +200,7 @@ def with_puppet_running_on(host, conf_opts, testdir = host.tmpdir(File.basename(
                                           'puppet.conf')
             lay_down_new_puppet_conf host, conf_opts, testdir
 
-            if host.use_service_scripts? && !service_args[:bypass_service_script]
-              bounce_service( host, host['puppetservice'], curl_retries )
-            else
-              puppet_master_started = start_puppet_from_source_on!( host, cmdline_args )
-            end
+            puppet_master_started = start_puppet_from_source_on!( host, cmdline_args )
```

Comment:
I think we need to keep the `bounce_service` call instead of using `start_puppet_from_source_on!`, as the latter runs `$ puppet master` which doesn't work on Puppet > 6:
https://github.com/puppetlabs/beaker-puppet/blob/9536cbb55a884c5a461156a406a69cda27967569/lib/beaker-puppet/helpers/puppet_helpers.rb#L286
We presumably don't need the `start_puppet_from_source_on!` method at all going forward since it primarily just runs `$ puppet master`